src/main/java/com/iexec/worker/feign/CustomCoreFeignClient.java

```diff
@@ -52,7 +53,7 @@ String login() {
     /*
      * How does it work?
      * We create an HttpCall<T>, T being the type of the response
-     * body and it can be Void. We send it along with the arguments
+     * body which can be Void. We send it along with the arguments
      * to the generic "makeHttpCall()" method. If the call was
      * successful, we return a ResponseEntity<T> with the response
      * body, otherwise, we return a ResponseEntity with the call's failure
```

Comment:
I'm not sure we still want this doc. It seems too much descriptive for what is effectively done there. It seems we are trying to describe the behavior of `makeHttpCall`, which isn't the place to do so.

Reply:
This is not a javadoc comment, it is a comment which goal is to describe how methods performing REST calls are implemented in `CustomCoreFeignClient` and how to resolve generic types depending on the underlying API.
I think it is a good idea to keep this comment here for the moment and to wait further rework before removing this.